packages/ra-ui-materialui/src/input/BooleanInput.js

```diff
@@ -25,38 +27,48 @@ export class BooleanInput extends Component {
             options,
             fullWidth,
             meta,
+            helperText,
             ...rest
         } = this.props;
 
         const { value, ...inputProps } = input;
+        const { touched, error } = meta;
 
         return (
-            <FormGroup className={className} {...sanitizeRestProps(rest)}>
-                <FormControlLabel
-                    htmlFor={id}
-                    control={
-                        <Switch
-                            id={id}
-                            color="primary"
-                            checked={!!value}
-                            onChange={this.handleChange}
-                            {...inputProps}
-                            {...options}
-                        />
-                    }
-                    label={
-                        <FieldTitle
-                            label={label}
-                            source={source}
-                            resource={resource}
-                            isRequired={isRequired}
-                        />
-                    }
-                />
-                {meta.error && (
-                    <FormHelperText error>{meta.error}</FormHelperText>
-                )}
-            </FormGroup>
+            <FormControl className={className} {...sanitizeRestProps(rest)}>
```

Comment:
applying the className to Formcontrol instead of FormGroup is technically a breaking change. Can you avoid that?